src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -265,7 +262,7 @@ void manualRegistrationToolBox::stopManualRegistration()
 
 void manualRegistrationToolBox::synchroniseMovingFuseView()
 {
-    if(this->parentToolBox()) //no need in Reg. workspace
+    if(selectorToolBox()) //no need in Reg. workspace
```

Comment:
Oh ok, the comment lost me. It means "// need to be tested in pipelines". Maybe we can change the comment.